doc/source/whatsnew/v0.16.0.txt

```diff
@@ -130,6 +130,54 @@ methods (:issue:`9088`).
     d    7
     dtype: int64
 
+- default behavior for HDF write functions with "table" format is now to keep rows that are all missing except for index. (:issue:`9382`)
+ 
+ Previously,
+ 
+  .. code-block:: python
+    In [1]: 
+     df_with_missing = pd.DataFrame({'col1':[0, np.nan, 2], 'col2':[1, np.nan, 3]})
+     df_with_missing.to_hdf('file.h5', 'df_with_missing', format = 't')
+     
+     df_without_missing = pd.DataFrame({'col1':[0, -1, 2], 'col2':[1, -1, 3]})
+     df_without_missing.to_hdf('file.h5', 'df_without_missing')
+     
+     print(pd.read_hdf('file.h5', 'df_with_missing'))
```

Comment:
you don't need print statements, paste the actual ipython output (which will have numbered In/Outs)